source/crud/crud.rst

```diff
@@ -1540,7 +1538,7 @@ Q: Where is write concern?
   However, it might be that a driver needs to expose write concern to a user per operation for various reasons. As noted before, it is permitted to specify this, along with other driver-specific options, in some alternative way.
 
 Q: How do I throttle unacknowledged writes now that write concern is longer defined on a per operation basis?
-  Some users used to throttle unacknowledged writes by using a write concern every X number of operations. The proper way to handle this on >= 2.6 servers is to use the bulk write API. Users working with servers < 2.6 should manually send a ``getLastError`` command every X number of operations if the driver does not support write concerns per operation.
+  Some users used to throttle unacknowledged writes by using an acknowledged write concern every X number of operations. Going forward, the proper way to handle this is by using the bulk write API.
```

Comment:
I was comfortable removing the last sentence in the original text, as that approach is exactly what is specified by the first sentence. If users decide they still want to use unacknowledged writes, for which the CRUD spec will allow legacy opcodes, they can still periodically use an acknowledged write concern. That will effectively interleave their legacy opcodes with write commands on 2.6+ and achieve the same throttling.